apluslms_roman/cli.py

```diff
@@ -365,20 +435,155 @@ def build_action(context):
     return result.code
 
 
+def init_action(context):
+    cwd = getcwd()
+    try:
+        config = ProjectConfig.find_from(cwd)
+        exit(1, _("A project configuration already exists at {}".format(config.path)))
+    except FileNotFoundError:
+        pass
+
+    if context.args.project_config:
+        file_ = context.args.project_config
+        if '.' not in file_:
+            is_recognized = False
+        else:
+            name, prefix = file_.split('.', 1)
```

Comment:
This `name, _delim, prefix = file_.rpartition('.')` would support filenames `foo.bar.yml`. Of course, `foo.bar` is not in DEFAULT_NAMES, thus following test would fail anycase. Just pointing out that when working with filename extensions `rpartition`, `rsplit` and `os.path.splitext`.

To note, current implementation does work, even though `name` and `prefix` will contain strange values in many cases when we get value `is_recognized = False`.

Reply:
Yeah I did consider that there might be several periods in the name, but since it wouldn't be any of the default names I decided to ignore it. Good to know about the filename methods though.